### PR TITLE
Remove UDP_stubs.h

### PR DESCRIPTION
**Pull request by cmcfarlen**
Followup to #9401

### CODE REVIEW

File: doc/developer-guide/api/types/TSSslSession.en.rst [hunk 1]
```diff
@@ -34,7 +34,6 @@ Synopsis
 
    .. member:: size_t len
 
-   .. member:: char bytes[TS_SSL_MAX_SSL_SESSION_ID_LENGTH]
```
Comment (bneradt) [inline doc/developer-guide/api/types/TSSslSession.en.rst:37]: This seems unrelated. Did you have to do this to make docs happy for some reason? If so, we should make it a separate PR. I'd like to know what introduced the change in behavior...

**Review by bneradt — CHANGES_REQUESTED** [review]

File: iocore/net/I_UDPPacket.h [hunk 2]
```diff
@@ -63,48 +63,9 @@ class UDPPacket
...
-/**
-   Create a new packet to be sent over UDPConnection. This actually
-   copies data from a buffer.
 
-
-   @param to  address of where to send packet
-   @param when ink_hrtime relative to ink_get_hrtime_internal()
-   @param buf if !nullptr, then len bytes copied from buf and made into packet.
-   @param len # of bytes to copy from buf
- */
```
Comment (bneradt) [inline iocore/net/I_UDPPacket.h:77]: Looks like these docs are being dropped in the diff.

**Review by bneradt — CHANGES_REQUESTED** [review]
Looks great. Thanks for cleaning up the build.

File: doc/developer-guide/api/types/TSSslSession.en.rst (hunk 1 above)
Comment (zwoop) [reply doc/developer-guide/api/types/TSSslSession.en.rst]: Yeh, this is my bad, I gave him a bad patch. This should not be done.

**Comment by zwoop** [comment]
Curious, does this fix the ```make check``` target on macOS?

**Comment by maskit** [comment]
> does this fix the make check target on macOS?

It works for me even without this change.

File: doc/developer-guide/api/types/TSSslSession.en.rst (hunk 1 above)
Comment (bneradt) [reply doc/developer-guide/api/types/TSSslSession.en.rst]: Gotcha.

**Comment by zwoop** [comment]
> > does this fix the make check target on macOS?
> 
> It works for me even without this change.

Weird, it fails on my mac and on the CI :/.

**Comment by cmcfarlen** [comment]
I tested on Mac, Linux, and the ci docker image.  I think it's good.  I would like to cleanup more Inline.cc as I don't believe it buys us anything anymore.

**Review by bneradt — APPROVED** [review]
Looks good.

**Comment by bneradt** [comment]
[approve ci]

File: iocore/net/I_UDPPacket.h (hunk 2 above)
Comment (zwoop) [reply iocore/net/I_UDPPacket.h]: Yeh, again my fault, and honestly I don't find those useful anyways. But yeh, we can ask Chris to restore them. One thing though, several of those "extern" declarations had no implementation anywhere, so they were there, documented, and not usable.

**Review by zwoop — CHANGES_REQUESTED** [review]
I think we need to address those removals that accidentally snuck in.

**Comment by cmcfarlen** [comment]
[approve ci]

**Comment by cmcfarlen** [comment]
> I think we need to address those removals that accidentally snuck in.

I restored the doc line and the doxygen comments. Was there something else?

**Review by zwoop — APPROVED** [review]
